src/app/checkout/step-1/step-1.component.js

```diff
@@ -11,18 +10,52 @@ let componentName = 'checkoutStep1';
 class Step1Controller{
 
   /* @ngInject */
-  constructor(){
-    this.contactType = 'individual';
+  constructor($q, cartService){
+    this.cartService = cartService;
+
+    this.init();
+    this.submitDetails = function(valid){
```

Comment:
If we are using classes, I think these functions should be methods outside the constructor. Like `init` is.